db/migrate/20170322052959_add_num_reviews_to_assignments.rb

```diff
@@ -0,0 +1,5 @@
+class AddNumReviewsToAssignments < ActiveRecord::Migration
+  def change
+    change_column :assignments, :num_reviews, :integer, :default => 3
```

Comment:
You changed the default value of the field num_reviews, so the class name should not be "Add...."